db/migrate/20160715124444_add_count_of_pension_pots.rb

```diff
@@ -0,0 +1,5 @@
+class AddCountOfPensionPots < ActiveRecord::Migration
+  def change
+    add_column :appointment_summaries, :count_of_pension_pots, :integer
```

Comment:
Yep, this was intentional as any default could give false data, and null allows then to skip the field if they don't know the answer.